app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -32,16 +32,13 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
       }
     }}
   >
-    {({ experiment, analysis }) => (
-      <>
-        <MonitoringLink {...experiment} />
-        {analysis?.show_analysis ? (
-          <AnalysisAvailable {...{ experiment, analysis }} />
-        ) : (
-          <AnalysisUnavailable {...{ experiment, error: analysis == null }} />
-        )}
-      </>
-    )}
+    {({ experiment, analysis }) =>
+      analysis?.show_analysis ? (
```

Comment:
This is the line @jodyheavener and I think needs a tweak. Looking at our mocked data here, is the fix as simple as changing this line to `analysis?.show_analysis && analysis?.overall ? (` or even just changing our check to `.overall` since @emtwo pointed out this should now always be true?

Reply:
yes, I was just going to say this! I just tested it out locally and adding that extra check here for `analysis?.overall` fixes the issue.

Reply:
For context, to test it locally, I created one of the experiments that has this issue in prod: `first-aboutnewtab-nightly-aa` using the admin. I made sure its status is set to complete. The viz API returns an object with `overall: null` so that check here makes it fall back to analysis unavailable.